src/azure-cli/HISTORY.rst

```diff
@@ -50,6 +50,7 @@ Release History
 * Fix #2531: Add convenience arguments for hub update.
 * Fix #8323: Add missing parameters to create storage custom endpoint.
 * Fix regression bug: Reverting the changes which overrides the default storage endpoint.
+* [BREAKING CHANGE] Removed --failover-region parameter from manual-failover. Now it will failover to assigned geo-paired secondary region.
```

Comment:
Please rebase dev branch and place your release not in the right position for next release.

Reply:
done